students/briggsm/session04/mb_trigram.py

```diff
@@ -1,54 +1,81 @@
+''''
+Matt Briggs Trigram Exercise
+Version 2.0 2017.11.19
+'''
+
 import random as rand
 
+
 def create_corpus(infile):
-    '''Input: a filename
-       Output: a list of words.'''
+    ''' Input: a filename of a text file
+        Output: a list of words from the text file.'''
     corpus = []
     with open(infile) as lines:
         lines.readline()
         for line in lines:
             line = line.strip()
             words = line.split(" ")
-            for i in range(len(words)):
-                corpus.append(words[i])
+            for word in enumerate(words):
```

Comment:
if you aren't uing the index, no need for enumerate:

`for word in words:`

works great.

and if you do need the index, then:

`for i, word in words:`

gives you the index and the list item as separate variables.